src/shared/types/focus-object/focusObjectTypes.ts

```diff
@@ -45,7 +45,7 @@ export type FocusLocation = BasicFocusObject & {
   type: 'location';
 };
 
-export type FocusVariant = BasicFocusObject & {
+export type FocusVariant = Omit<BasicFocusObject, 'location'> & {
```

Comment:
Let's keep it. We'll see later if we need it. I suspect we might.

Reply:
✅